big-practice/src/js/helpers/show-message.helper.js

```diff
@@ -0,0 +1,13 @@
+export class Document {
```

Comment:
First, file name is not correct

Second, class should be added postfix Helper

Reply:
updated!